### PR TITLE
Fixed some errors in the code.

### PR DESCRIPTION
**Pull request by kwcckw**
1. Fixed typo in BrightnessTexturizeAugmentation.
2. Fixed  channel mismatch in LowInkBlobsAugmentation.
3. Added evaluation to prevent error when there is no paper image in the paper directory.

### CODE REVIEW

File: src/augraphy/augmentations/dirtyrollers.py [hunk 1]
```diff
@@ -66,6 +66,8 @@ def __call__(self, data, force=False):
                 image = cv2.rotate(image, cv2.ROTATE_90_CLOCKWISE)
 
             image = cv2.cvtColor(image, cv2.COLOR_BGR2GRAY)
+            
+            # where is this transform?
```
Comment (kwcckw) [inline src/augraphy/augmentations/dirtyrollers.py:70]: @tbrettallen so we should remove the entire self.transform in dirtyrollers augmentation? Right now i can see it is still in the code, and couldn't locate the definition of self.transform.

Comment (proofconstruction) [reply src/augraphy/augmentations/dirtyrollers.py:70]: This `transform` method call is left over from before [this commit](https://github.com/sparkfish/augraphy/commit/ea0970932b49288b9893e728d8c4442693b825e4) when it was removed. Probably is why the default pipeline has `DirtyRollersAugmentation(p=0.0)` set in the `post_phase`; it doesn't work anymore. I'll make an issue for this.

**Review by proofconstruction — APPROVED** [review]
The changes look good, approved.

Notes:
---
My PR including [commit a7b4e7a](https://github.com/proofconstruction/augraphy/commit/a7b4e7ad9dbf8ec318902fe2119142bd5d4a814d) didn't get merged yet, but I corrected the `np.array` -> `np.zeros` problem there. This bug was introduced (by me! thanks for catching it) as part of removing dependencies on `numpy.random`.

The changes in LowInkBlobs look good, but we're moving that out into the library module at `augmentations/lib.py`, since LowInkBlobs and DustyInk are meant for Letterpress. We can fix this later.

The additions to PaperFactory are also good, but I think we should soon find a better way (throwing exceptions) to deal with these kinds of errors than printing to the console.

DirtyRollers is actually broken right now, see issue #36.

**Comment by kwcckw** [comment]
> The changes in LowInkBlobs look good, but we're moving that out into the library module at `augmentations/lib.py`, since LowInkBlobs and DustyInk are meant for Letterpress. We can fix this later.

Right, i would think move to lib.py is better, since actually i need to use those functions in some other augmentations too.

> 
> The additions to PaperFactory are also good, but I think we should soon find a better way (throwing exceptions) to deal with these kinds of errors than printing to the console.
> 

Yes, probably in the future we can introduce certain level of logging at higher level, but that probably can be done later. Using print should be the simplest approach for now.

Anyway, thanks for the feedback.

**Pull request merged**